02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Nice! The mapping from bond id's to Cylinder geometry is very clean and simple.</p><p> </p><p>We could separate the rendering out into another cell to differentiate it from the bond geometry. Let's also remove the unused material color. Do you prefer the thick box, or should we make it thinner?</p><p> </p><p>If we do change from unwrapping to removing bonds, the particles might be close enough together to look nice with path tracing and cloudy or lightbox lighting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Once I get the unwrapping figured out, I will play around with some lighting to make things really look good :)</p>

Reply:
Still need to figure out lighting/materials, but other than that I think it is looking pretty good.